src/main/java/app/linguistai/bmvp/service/AccountService.java

```diff
@@ -72,20 +72,9 @@ public RLoginUser login(QUserLogin user) throws Exception {
             final String accessToken = jwtUtils.createAccessToken(userDetails);
             final String refreshToken = jwtUtils.createRefreshToken(userDetails);
 
-            // If login is successful, check whether to increase user streak or not
-            // ALSO IN MESSAGE SERVICE: userStreakService.updateUserStreak(dbUser.getEmail());
-            try {
-                userStreakService.updateUserStreak(dbUser.getEmail());
-            }
-            catch (Exception e1) {
-                // Intentionally not thrown to not cause login exception for users without UserStreak
-                System.out.println(ExceptionLogger.log(e1));
-            }
+            initiateUserSession(dbUser.getEmail());
```

Comment:
Although it might seem redundant, adding `this.` prefix to in-service private function calls can improve the readability and maintainability of the code

Reply:
Added